src/Generators/Microsoft.Gen.Logging/Parsing/Parser.cs

```diff
@@ -378,12 +398,26 @@ public IReadOnlyList<LoggingType> GetLogTypes(IEnumerable<TypeDeclarationSyntax>
                 keepMethod = false;
             }
 
-            if (templatesWithAtSymbol.Count > 0)
+            TemplateExtractor.ExtractTemplates(message, lm.Templates);
+
+#pragma warning disable EA0003 // Use the character-based overloads of 'String.StartsWith' or 'String.EndsWith'
```

Comment:
Why can't we use it?

Reply:
The overload doesn't exist. It's an analyzer bug, it shouldn't be recommending it.